ui-grid/ui-grid-tests.ts

```diff
@@ -54,7 +61,8 @@ columnDef.headerCellFilter = 'currency:$';
 columnDef.headerCellTemplate = '<div class="yoshi"></div>';
 columnDef.headerTooltip = false;
 columnDef.headerTooltip = 'The Tooltip';
-columnDef.headerTooltip = (col: uiGrid.IGridColumn) => {
+columnDef.headerTooltip = (col) => {
+    //type of col flows in properly
```

Comment:
I suppose the only downside is redundancy, since the compiler already knows what type it is.  If you are using generics, for example, adding the non-generic type annotation makes your autocompletion less useful since `IGridColumn` uses the `<any>` type for the generic argument.